scRNA-seq/01-normalizing_scRNA-seq.Rmd

```diff
@@ -61,28 +62,222 @@ and could use these same steps to filter and normalize the data.
   
 ```{r Import Data}
 # Read in the data
-sc_data <- readr::read_tsv(file.path("data", "darmanis_counts.tsv"), 
-                           progress = FALSE) %>%
-  as.data.frame() %>%
-  # We will need our genes as the rownames for setting up our data with Seurat
-  tibble::column_to_rownames("gene") 
+sc_data <- readr::read_tsv(file.path("data", "unfiltered_darmanis_counts.tsv"), 
```

Comment:
It is still a subset of the full dataset, but it is more than the original number because I added back in the low percentage mapped reads that I originally filtered out so the participants can filter those samples out themselves.